kthresher.bash_completion

```diff
@@ -15,31 +19,30 @@ genOpts()
   local IFS
   local not_used=""
   # options in pairs
-  local opts="--help,-h --config,-c --dry-run,-d --headers,-H --keep,-k --purge,-p --show-autoremoval,-s --verbose,-v --version,-V"
+  local opts="--help,-h --config,-c --dry-run,-d --headers,-H --keep,-k --purge,-p "
+  opts+="--show-autoremoval,-s --verbose,-v --version,-V"
   for i in ${opts} ; do
-    OLDIFS=$IFS
+    OLDIFS=${IFS}
     # set IFS to split our pairs
     IFS=","
-    set -- $i
+    set -- ${i}``
     # reset IFS to original value
-    IFS=$OLDIFS
-    result_one=$(containsElement "${1}" "${COMP_WORDS[@]}" ; echo $?)
-    result_two=$(containsElement "${2}" "${COMP_WORDS[@]}" ; echo $?)
-    # if neither exists in COMP_WORDS, both short and long opt are valid for use
-    if [[ "${result_one}" != 0 ]] && [[ "${result_two}" != 0 ]]; then
-      not_used="${not_used} ${1}"
-      not_used="${not_used} ${2}"
+    IFS=${OLDIFS}
+    if ! $( containsElement "${1}" "${COMP_WORDS[@]}" ) && \
+       ! $( containsElement "${2}" "${COMP_WORDS[@]}" ); then
+         not_used="${not_used} ${1}"
+         not_used="${not_used} ${2}"
```

Comment:
The comments I did was on the if statement, I see that has been changed as suggested, great!, minor think with `&&` there is no need for `\` at the end of the line when having `&&`, `||` or `|` so one less char there :) .

What I missed in the first review is the assignment of `not_used`, is there any reason why two assignments are after each other? instead of appending both elements at once? e.g.
```bash
not_used="${not_used} ${1} ${2}"
```